Transformers/ServiceNowFormDataTransformer.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace EMS\SubmissionBundle\Transformers;
+
+class ServiceNowFormDataTransformer extends AbstractFormDataTransformer
+{
+    public function transform(): array
+    {
+        $data = $this->form->getData();
+
+        return $this->mapNewFieldTimeFormat($data);
```

Comment:
I like the "map" in your name. Very explicit! (y) 
But I think the "NewField" is not needed, as we are mapping on the FormData (FormDataTransformer). If you are okay with it I propose "mapTimeFormat"

Reply:
I would also make this a one liner, the $data variable is not really used